lib/msal-browser/src/utils/BrowserUtils.ts

```diff
@@ -15,7 +15,7 @@ export class BrowserUtils {
      * Used to redirect the browser to the STS authorization endpoint
      * @param {string} urlNavigate - URL of the authorization endpoint
      */
-    private navigateWindow(urlNavigate: string): void {
+    static navigateWindow(urlNavigate: string): void {
         throw new Error("Method not implemented.");
```

Comment:
Do we need this method? If so, why not implement?

Reply:
Will be implemented in a future PR - just added it for the sake of the PR. I can remove it later.